app/routes.py

```diff
@@ -13,7 +13,7 @@
 
 @app.route('/')
 def index():
-    return render_template('index.html', title='App Name')
+    return render_template('index.html', title='Literapy')

```

Comment:
Since you changed this in both branches you will have a merge conflict.

Reply:
EDIT: Just checked, you should be fine to merge. Since the changes are exactly the same, no merge conflicts will arise :-)